microshift_storage/understanding-persistent-storage-microshift.adoc

```diff
@@ -23,6 +23,13 @@ include::modules/storage-persistent-storage-reclaim.adoc[leveloffset=+2]
 
 include::modules/storage-persistent-storage-pv.adoc[leveloffset=+1]
 
+ifdef::microshift[]
+[id="additional-resources"]
+[role="_additional-resources"]
+.Additional resources
+* For a list of commonly supported mount options, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_file_systems/mounting-file-systems_managing-file-systems#common-mount-options_mounting-file-systems[Common mount options]
```

Comment:
I can't remember if Microshift maybe had an exemption from this, but usually "Additional resources" items should just be the link with no extra text / sentence surrounding it ([repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)).

Reply:
Hi @ShaunaDiaz,
 On a quick check across the Microshift docs for "Additional resources" section, it seems that they all comply with the repo guidelines that @adellape mentioned? So may I request you to implement this suggestion unless there are strong reasons not to?

Reply:
We do have links this way in MicroShift. I do also think this is outside the scope of this PR. But I will update these as requested.